compiler/src/dotty/tools/dotc/classpath/FileUtils.scala

```diff
@@ -20,6 +20,10 @@ object FileUtils {
     def isClass: Boolean = !file.isDirectory && file.hasExtension("class") && !file.name.endsWith("$class.class")
       // FIXME: drop last condition when we stop being compatible with Scala 2.11
 
+    def isClassExtension: Boolean = file.hasExtension("class")
```

Comment:
```suggestion
    def hasClassExtension: Boolean = file.hasExtension("class")
```

Reply:
This could be used in the other methods such as `isClass`.

Reply:
done